ePanchayat.API/CommonUtils/Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ePanchayat.API.CommonUtils
+{
+    public struct Constants
+    {
+        public const string ConnectionString = @"Data Source =MARU\MARU; Initial Catalog =ePanchayat;User ID = pmaru; Password=pmaru@123;";
```

Comment:
You should never has username and password in connection string added like this. we can move it to a secure location and read from there.

Reply:
With initial set-up we are okay for now, once set-up is completed we will address such vulnerability.